packages/react-core/src/InAppMessaging/utils/handleMessageAction.ts

```diff
@@ -21,7 +20,7 @@ const handleMessageAction = ({
   logger.info(`Handle action: ${action}`);
 
   if (action === 'LINK' || action === 'DEEP_LINK') {
-    if (!isString(url)) {
+    if (typeof url !== 'string') {
```

Comment:
Nit: Fwiw, it is possible to do something weird like
```ts
const url = new String("http://url.com");
```
and the `typeof` url would be `object` instead of `string`. I think `isString` checks for this as well